config/routes.rb

```diff
@@ -30,6 +31,12 @@
         post 'reprocess'
       end
     end
+    resources :device_logs
+    resources :device_commands do
```

Comment:
You are missing a `only: [:index]` for `:device_commands` and `only: [:index, :show, :create]` for `:device_logs`.